cocos/2d/assets/sprite-atlas.ts

```diff
@@ -101,7 +101,7 @@ export class SpriteAtlas extends Asset {
         const frames: Array<SpriteFrame | null> = [];
         const spriteFrames = this.spriteFrames;
 
-        for (const key of Object.keys(spriteFrames)) {
+        for (const key in spriteFrames) {
```

Comment:
The same as above.

Reply:
Replied. there is no differences for plain objects. spriteFrames is a plain object.

Reply:
Ok. Please make sure all the modification is applied to plain object.